src/js/pre-logic.js

```diff
@@ -113,13 +182,13 @@ function crunchSprite(parentState) {
     if (isAnEdge) {
       switch(direction) {
       case 'left':
-        return {...state, x: parentState.board.rows};
+        return {...state, x: parentState.board.rows - 1};
```

Comment:
yes, I want to hoist the rows. If I remove the higher order function the testing will be impossible because currentState won't be explicitly defined/passed in